index.html

```diff
@@ -27,6 +27,13 @@ <h2 style="color: steelblue">Connect with me!</h2>
             </ul>
         </section>
      </center>
+
+     <script src="js/index.js">
+       <footer>
+
+        </footer>
+     </script>
```

Comment:
```suggestion
     <script src="js/index.js"></script>
     <div class=".footer"></div>
```

- If a `script` tag has a `src`, it shouldn't contain anything. (If it doesn't have a `src`, the contents should be javascript, not html.)
- `'.footer'` will select elements whose class is `footer`... If you want the `<footer>` element, remove the `.` prefix in the the selector.